.github/workflows/ci_windows_mu4.yml

```diff
@@ -53,7 +53,10 @@ jobs:
       shell: bash
       env:
         pull_request_title: ${{ github.event.pull_request.title }}
-      run: |
+        SENTRY_SERVER_MU4_KEY: ${{ secrets.SENTRY_SERVER_MU4_KEY }}
+        SENTRY_SERVER_SANDBOX_KEY: ${{ secrets.SENTRY_SERVER_SANDBOX_KEY }}
+        SENTRY_PROJECT: ${{ github.event.inputs.sentry_project }}
+       run: |
```

Comment:
```
run: |
```
This change needs to be removed, it leads to an error (I fixed it for the master)

Reply:
Sorry, fixed too now